macros/sql/weekdays_between_dates.sql

```diff
@@ -0,0 +1,31 @@
+--code source: https://discourse.looker.com/t/how-to-count-only-weekdays-between-two-dates/3345
```

Comment:
Referencing the source is good, but we can't rely on that link working indefinitely, and the logic here is pretty arcane.  I'd suggest incorporating some of the explanation from that article here, such as commenting each block of logic with the relevant heading from that article, like:
```sql
    -- Subtract one weekend for each complete week.
    (
        floor (
            date_diff('day', {{start}}, {{stop}}) / 7
        ) * 2
    )
```
And so on.

Reply:
Added comments for each step